composer.json

```diff
@@ -53,5 +54,8 @@
         }
     },
 
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
But it doesn't solve the problem that if Puli is provided in an extension (official or not), it would indeed be hard to solve the problem of extending these (you would actually need to load the extension through a normal way ?)

Or what could be done, instead of relying on these, would be to support something like a composer package type, which would be loaded automatically ?